test/team_statistics_test.rb

```diff
@@ -0,0 +1,27 @@
+require "./lib/team_statistics"
```

Comment:
@Jonathan-M-Wilson These are tests for the inherited class methods from StatTracker. Notice how they all still pass even though your team_statistics.rb doesn't have any code in it! WooHoo!